packages/storage/src/database/base/basic.ts

```diff
@@ -73,12 +72,6 @@ export abstract class ModelBasic<T extends Document> {
   ): Promise<void> {
     const fieldName = `${zkDatabaseConstant.systemIndex}_${Object.keys(indexSpec).join('_')}`;
 
-    const { error } = JOI_ZKDB_FIELD_NAME.validate(fieldName);
```

Comment:
I think we should not delete this JOI, We should update the JOI_ZKDB_FIELD_NAME
allow _ at first is valid
```ts
/^[a-zA-Z_][a-zA-Z0-9_]{1,128}$/
```

Reply:
fixed by validating the index spec